CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+Guidelines for contributing to Infinispan Management Console 
+====
+
+Contributions from the community are essential in keeping Infinispan Management Console strong and successful.
+
+This guide focuses on how to contribute back to Infinispan Management Console using GitHub pull requests.
+If you need help with cloning, compiling or setting the project up in an IDE please refer to
+[this page](http://infinispan.org/docs/stable/contributing/contributing.html).
+
+## Legal
+
+All original contributions to Infinispan Management Console are licensed under the
```

Comment:
The Console must be ASL, not LGPL.

Reply:
@tristantarrant see below the updated text